CHANGELOG/CHANGELOG-1.8.md

```diff
@@ -0,0 +1,100 @@
+- Rename Windows script Start.ps1 to Start-AntreaAgent.ps1, and rename Stop.ps1 to Stop-AntreaAgent.ps1. ([#3904](https://github.com/antrea-io/antrea/pull/3904), [@wenyingd])
+- Unify NodePortLocal behavior across Linux and Windows. Linux agents now support allocating different nodeports for different protocols when the podports are the same. ([#3936](https://github.com/antrea-io/antrea/pull/3936), [@XinShuYang])
+- Antrea IPAM now uses the name of the uplink interface to name the host internal port, the uplink interface will be renamed with a `~` suffix, e.g. `eth0~`. ([#3938](https://github.com/antrea-io/antrea/pull/3938), [@gran-vmv])
+- Send Neighbor Advertisement messages after creating Pods in IPv6 cluster. ([#3998](https://github.com/antrea-io/antrea/pull/3998), [@gran-vmv])
+- Add columns of kubectl outputs for Multi-cluster types. ([#3923](https://github.com/antrea-io/antrea/pull/3923), [@jianjuns])
+- Add IP version and prefix length validation to IPPool CRD manifest. ([#3897](https://github.com/antrea-io/antrea/pull/3897), [@jianjuns])
+- Add an output formatter "raw" to better display multi-line string responses for antctl. ([#3589](https://github.com/antrea-io/antrea/pull/3589), [@Atish-iaf])
+- Add new ports to network requirement doc. ([#4063](https://github.com/antrea-io/antrea/pull/4063), [@luolanzone])
+- Windows OVS installation script now installs required SSL library if missing. ([#4029](https://github.com/antrea-io/antrea/pull/4029), [@XinShuYang])
+- Upgrade whereabouts CNI to v0.5.4 and align secondary network IPAM with additional pluginArgs. ([#3987](https://github.com/antrea-io/antrea/pull/3987), [@arunvelayutham])
+- Remove Theia related files. ([#4048](https://github.com/antrea-io/antrea/pull/4048), [@dreamtalen])
+<!-- 
+- Use hostNetwork for Multi-cluster controller. ([#3965](https://github.com/antrea-io/antrea/pull/3965), [@luolanzone])
+- Update ClusterClaim to v1alpha2. ([#3755](https://github.com/antrea-io/antrea/pull/3755), [@bangqipropel])
+- Update GatewayIPPrecedence to support "external/internal" option for the Multi-cluster feature. ([#3930](https://github.com/antrea-io/antrea/pull/3930), [@luolanzone])
+- Disable metrics API and change the health binding address port to 8080 for the Multi-cluster feature. ([#4101](https://github.com/antrea-io/antrea/pull/4101), [@luolanzone])
+- Improve CRD validation for the Multi-cluster feature. ([#4062](https://github.com/antrea-io/antrea/pull/4062) [#4090](https://github.com/antrea-io/antrea/pull/4090), [@luolanzone])
+- Cleanup ResourceExport if exported Service has no available Endpoints. ([#4056](https://github.com/antrea-io/antrea/pull/4056), [@luolanzone])
+- Add controller for leader cluster to garbage collect MemberClusterAnnounce for the Multi-cluster feature. ([#4054](https://github.com/antrea-io/antrea/pull/4054), [@hjiajing])
```

Comment:
For this one, we can just append the PR to the next one.

Reply:
merged